packages/cli/src/collect/node-runner.js

```diff
@@ -138,7 +138,7 @@ class LighthouseRunner {
     while (attempts.length < 3) {
       try {
         return await this.run(url, options);
-      } catch (err) {
+      } catch (/** @type {any} */ err) {
```

Comment:
if these are always going to be cast to `any`, there's not really any point to the check and it's probably just better to disable it with `"useUnknownInCatchVariables": false` to avoid the noise and potential future issues with type casts. That's what we [do in Lighthouse](https://github.com/GoogleChrome/lighthouse/blob/82772f93645989c9c250c78a0ad10d55f3372beb/tsconfig-base.json#L18).